internal/network/command_network_link_endpoint_list.go

```diff
@@ -0,0 +1,62 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type listNetworkLinkEndpointOut struct {
+	Id                 string `human:"ID" serialized:"id"`
+	Name               string `human:"Name" serialized:"name"`
+	Network            string `human:"Network" serialized:"network"`
+	Description        string `human:"Description,omitempty" serialized:"description,omitempty"`
+	NetworkLinkService string `human:"Network Link Service" serialized:"network_link_service"`
+	Phase              string `human:"Phase" serialized:"phase"`
+}
```

Comment:
Looks like this is the same struct as `describe`, just without the "Environment" field. You can use the `Filter()` function so you don't need to redeclare a new struct.

Reply:
Updated!